src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -261,6 +267,8 @@ def __init__(
                 quantization_config.format, config=quantization_config
             )
 
+    # ----- used by hf quantizer ----- #
```

Comment:
I like this call out! we should make this a convention to call out methods used by integrations downstream